jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/VirtualThreads.java

```diff
@@ -38,7 +39,11 @@ private static Executor probeVirtualThreadExecutor()
     {
         try
         {
-            return (Executor)Executors.class.getMethod("newVirtualThreadPerTaskExecutor").invoke(null);
+            Class<?> builderClass = Class.forName("java.lang.Thread$Builder");
+            Object threadBuilder = Thread.class.getMethod("ofVirtual").invoke(null);
+            threadBuilder = builderClass.getMethod("name", String.class, long.class).invoke(threadBuilder, "jetty-vt-", 0L);
```

Comment:
Like all of our other Thread implementations we make the name configurable.
Can you do this with your PR?

Reply:
Example from the non-virtual thread implementation in our code ...

QueuedThreadPool is a ThreadFactory.

https://github.com/jetty/jetty.project/blob/3aed62e4959bb8c01f5975fe81e078e3ff626126/jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java#L902-L913

Reply:
@joakime how exactly should this be made configurable? Can you please link me to an existing implementation? I will then follow the same approach here. 

The other two instances I could find both use a non-configurable prefix:
1. https://github.com/jetty/jetty.project/blob/jetty-12.0.x/jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java#L103
2. https://github.com/jetty/jetty.project/blob/jetty-12.0.x/jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/ExecutorThreadPool.java#L44

Reply:
heh, I think we crossed posted comments (see mine https://github.com/jetty/jetty.project/pull/11355#discussion_r1471265495)

The `QueuedThreadPool` is a `ThreadFactory` and has an implementation of `newThread(Runnable)` that shows the non-virtual thread name setup there.

Reply:
@joakime as this is the default, I can live with hardcoded names.
Users that want different configuration and names can set their own virtual thread `Executor`.